packages/components/src/Badge/Badge.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { createCssClassNames } from '@ibexa/ids-core/helpers/cssClassNames';
+
+import type { BadgeProps } from './Badge.types';
+
+const MAX_BADGE_VALUE = 99;
+
+const formatBadgeValue = (value: number): string => {
+    return value > MAX_BADGE_VALUE ? '99+' : value.toString();
+};
+
+const Badge = ({ value, size = 'medium', className = '' }: BadgeProps) => {
```

Comment:
I'm disapointed by eslint. :C Could you order these in alphabetical order, pretty please? :pleading_face: